cache-config/testing/ort-tests/t3c-apply-unset-update_test.go

```diff
@@ -26,48 +25,65 @@ import (
 	"github.com/apache/trafficcontrol/lib/go-tc"
 )
 
+const cacheHostName = `atlanta-edge-03`
+const cmdUpdateStatus = `update-status`
+
+func verifyUpdateStatus(t *testing.T) {
```

Comment:
As above, I would've expected sub-tests to contain their setup.

This whole test `TestT3cUnsetsUpdateFlag` is really just 1 test, I'm not sure sub-tests make sense here. The block `// verify update status is now true` could be considered a sub-test, but it's really just verifying the setup, the purpose of the test is to make sure un-setting in t3c itself works; this isn't the place to test t3c-update or TO endpoints.

Reply:
I just wanted to get rid of the anonymous block scoping; if you think it's fine to do that without an equivalent like a function call I'm happy to just make the simpler change.

Reply:
What's the issue with the blocks? They serve to isolate the code, both for readability, and to isolate variables to prevent accidental reuse bugs.

Functions that aren't sub-tests would be fine IMO. Though I feel like separate functions in a different location would reduce readability here and make it harder for me to work with the code. But readability is relative, as always.

Reply:
I always find those anonymous blocks confusing. You may recall the first time I ever saw them; it was in one of your PRs and I had to ask what that even did. In retrospect I could've guessed, but it's still fairly unusual.

The difference between an anonymous block scope and a call into a named function block in terms of readability largely comes down to familiarity, IMO, since a good function name can have an equivalent in the form of a comment.

I think most people would agree with me that an anonymous block is uncommon, and I also think it's easy to miss, especially as the containing function grows larger with different levels of scope nesting; the beginning and end of such blocks are a single character that simply escape notice, so even while making these changes I found myself backtracking because "I swear there are no new variables on the left of this `:=`". In fact, as I'm looking at this file, I see one such block escaped my notice, when it was absolutely my intention to do away with those, which is the best illustration of the issue take with them that I could possibly imagine.

I think a closure function call would make such things more apparent, for whatever that opinion is worth (it comes from experience with JS I'm sure, and that's not exactly an ecosystem to emulate), like:
```go
func(){
    // Code
}()
```
instead of
```go
{
    // Code
}
```
just harder to miss the extra symbols there.

In general, though, I'm of the opinion that if a section of code has responsibility isolated from the surrounding block and requires its own scope for the best enforcement of that isolation, then it meets the definition of a function, and functions should have names that convey what they do.

With all that being said

> Functions that aren't sub-tests would be fine IMO

I'll do that, then